hindi/swami-vivekananda/swami-vivekananda brief introduction.md

```diff
@@ -0,0 +1,4 @@
+﻿स्वामी विवेकानन्द (जन्म: १२ जनवरी,१८६३ - मृत्यु: ४ जुलाई,१९०२) वेदान्त के विख्यात और प्रभावशाली आध्यात्मिक गुरु थे।
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.